### PR TITLE
Various CS fixes (Trac 53359)

### PR DESCRIPTION
**Pull request by jrfnl**
Trac ticket: https://core.trac.wordpress.org/ticket/53359

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/script-loader.php [hunk 1]
```diff
@@ -221,9 +221,9 @@ function wp_default_packages_scripts( $scripts ) {
 	$suffix = wp_scripts_get_suffix();
 
 	// Expects multidimensional array like:
-	//	'a11y.js' => array('dependencies' => array(...), 'version' => '...'),
-	//	'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
-	//	'api-fetch.js' => array(...
+	//  'a11y.js' => array('dependencies' => array(...), 'version' => '...'),
+	//  'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
+	//  'api-fetch.js' => array(...
```
Comment (hellofromtonya) [inline src/wp-includes/script-loader.php:226]: Shouldn't this be in multiline comment format?

Comment (jrfnl) [reply src/wp-includes/script-loader.php:226]: I agree that would be better, though that would be a different fix, outside the scope of this atomic commit.
Also: WP Core does not scan against the `WordPress-Docs` standard at all yet (and rightfully so as they are nowhere near good enough yet).

**Comment by hellofromtonya** [comment]
@jrfnl Is this PR complete and ready for code review?

**Comment by jrfnl** [comment]
@hellofromtonya There will always be more fixes to make, but the fixes as pulled here are ready for commit as far as I'm concerned.

File: src/wp-includes/blocks/calendar.php [hunk 2]
```diff
@@ -24,10 +24,8 @@ function render_block_core_calendar( $attributes ) {
 			strpos( $permalink_structure, '%monthnum%' ) !== false &&
 			strpos( $permalink_structure, '%year%' ) !== false
 		) {
-			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited
 			$monthnum = $attributes['month'];
-			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited
-			$year = $attributes['year'];
+			$year     = $attributes['year'];
 		}
```
Comment (jrfnl) [inline src/wp-includes/blocks/calendar.php:29]: The E2E tests are failing on the changes in this file.

I imagine this is due to the blocks code being maintained in a separate repo. In that case, the particular commit containing this change can be ignored.

Note: the `WordPress.WP.GlobalVariablesOverride` sniff is not active in WP Core as it is intended only for plugins and themes, so this ignore annotation really doesn't make sense (for Core).

**Review by hellofromtonya — APPROVED** [review]
Batch approval of the commits to this approval point ✅

File: src/wp-includes/blocks/calendar.php (hunk 2 above)
Comment (peterwilsoncc) [reply src/wp-includes/blocks/calendar.php:29]: > The E2E tests are failing on the changes in this file.

Are you able to remove these changes from 26ce822d20259762c6205f951a0cdbc1fb8deb55 and force push, just to be certain all other tests are passing as expected? 

I'm sure they are but it's nice to be certain.

Comment (jrfnl) [reply src/wp-includes/blocks/calendar.php]: Of course, not a problem. Rebased on current `master`, undid this particular change and force pushed.

Comment (jrfnl) [reply src/wp-includes/blocks/calendar.php]: All 🟢

**Comment by jrfnl** [comment]
I've rebased the remaining commits in this PR to solve the merge conflicts and make the patches commitable again.

File: phpcs.xml.dist [hunk 3]
```diff
@@ -231,23 +231,18 @@
...
-				<element value="WP_Ajax_UnitTestCase"/>
-				<element value="WP_Canonical_UnitTestCase"/>
-				<element value="WP_Test_REST_TestCase"/>
-				<element value="WP_Test_REST_Controller_Testcase"/>
-				<element value="WP_Test_REST_Post_Type_Controller_Testcase"/>
-				<element value="WP_XMLRPC_UnitTestCase"/>
+				<element value="WP_Test_XML_TestCase"/>
+
+				<!-- Test case parent classes outside of the "includes" folder. -->
+				<element value="Tests_Query_Conditionals"/>
```
Comment (hellofromtonya) [inline phpcs.xml.dist:239]: Making a note for anyone reviewing:

Why is this one in the list? It's serving 2 roles:

- Test case as `Tests_Query_VerbosePageRules` extends it
- and a test class as it has tests within it

**Review by hellofromtonya — APPROVED** [review]
Re-reviewed new changes. All look good and ready for `commit` 👍

**Comment by jrfnl** [comment]
Closing as fully committed via a variety of changesets.

**Pull request closed without merging**